config/crd/bases/stargate.k8ssandra.io_stargates.yaml

```diff
@@ -1104,7 +1104,7 @@ spec:
               containerImage:
                 default:
                   repository: stargateio
-                  tag: v1.0.66
+                  tag: v1.0.67
```

Comment:
Same.

Reply:
yeah, I think my previous PR which upgraded stargate hadn't rebuilt the CRDs manifests :( 
That's why it's here.
I'll ninja fix this